README.md

```diff
@@ -90,6 +93,8 @@ above.
   1. edit `/etc/openstack_deploy/user_extras_variables.yml` to add credentials
   2. run the MaaS setup plays:
      `cd /opt/rpc-openstack/rpcd/playbooks && openstack-ansible setup-maas.yml`
+  3. run the MaaS verify play:
```

Comment:
optionally, we include the verify play with retries and delays at the end of the setup play, similar to how I did in the upgrade play.  That might be more straight forward.  Thoughts?